vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -268,6 +268,7 @@
   - name: Set expected ironic image based containers
     set_fact:
       ironic_image_containers:
+        - ironic
```

Comment:
Since this is touching an upgrade, we have to run tests that checks these changes